README.md

```diff
@@ -237,469 +103,115 @@ You need to copy the following files:
-- Emil Djupfeld (egladil) - macOS port (0.4 - 0.6)
-- Simon Sasburg (HackyKid) - Bug fixer (0.4 - 0.4.5)
-- Ludvig Strigeus (ludde) - Original author of OpenTTD, main coder (0.1 - 0.3)
-- Cian Duffy (MYOB) - BeOS port / manual writing (0.1 - 0.3)
-- Petr Baudiš (pasky) - Many patches, newgrf support, etc. (0.3 - 0.3)
-- Benedikt Brüggemeier (skidd13) - Bug fixer and code reworker (0.6 - 0.7)
-- Serge Paquet (vurlix) - 2nd contributor after ludde (0.1 - 0.3)
-
-### Thanks to:
-
-- Josef Drexler - For his great work on TTDPatch.
-- Marcin Grzegorczyk - For his TTDPatch work and documentation of Transport Tycoon Deluxe internals and track foundations
-- Stefan Meißner (sign_de) - For his work on the console
-- Mike Ragsdale - OpenTTD installer
-- Christian Rosentreter (tokai) - MorphOS / AmigaOS port
-- Richard Kempton (RichK67) - Additional airports, initial TGP implementation
-- Alberto Demichelis - Squirrel scripting language
-- L. Peter Deutsch - MD5 implementation
-- Michael Blunck - For revolutionizing TTD with awesome graphics
-- George - Canal graphics
-- Andrew Parkhouse (andythenorth) - River graphics
-- David Dallaston (Pikka) - Tram tracks
-- All Translators - For their support to make OpenTTD a truly international game
-- Bug Reporters - Thanks for all bug reports
-- Chris Sawyer - For an amazing game!
+
+## 1.5) Add-on content / mods
+
+OpenTTD features multiple types of add-on content, which modify gameplay in different ways.
+
+Most types of add-on content can be downloaded within OpenTTD via the 'Check Online Content' button in the main menu.
+
+Add-on content can also be installed manually, but that's more complicated; the [OpenTTD wiki](https://wiki.openttd.org/OpenTTD) may offer help with that.
+
+### 1.5.1) AI opponents
+
+OpenTTD comes without AI opponents, so if you want to play with AIs you have to download them.
+
+The easiest way is via the 'Check Online Content' button in the main menu.
+
+You can select some AIs that you think are compatible with your playing style.
+
+AI help and discussions may also be found in the [AI section of TT-Forums](https://www.tt-forums.net/viewforum.php?f=65).
+
+### 1.5.2) Scenarios and height maps
+
+Scenarios and heightmaps can be added via the 'Check Online Content' button in the main menu.
+
+### 1.5.3) NewGRFs
+
+A wide range of add-content is available as NewGRFs, including vehicles, industries, stations, landscape objects, town names and more.
+
+NewGRFs can be added via the 'Check Online Content' button in the main menu.
+
+See also the wiki [guide to NewGRFs](https://wiki.openttd.org/NewGRF) and [TT-Forums graphics development section](https://www.tt-forums.net/viewforum.php?f=66).
+
+### 1.5.4) Game scripts
+
+Game scripts can provide additional challenges or changes to the standard OpenTTD gameplay, for example setting transport goals, or changing town growth behaviour.
+
+Game scripts can be added via the 'Check Online Content' button in the main menu.
```

Comment:
Very very repetitive "can be added via the ..." Put into parent section.

Reply:
It is very repetitive, but the information under each sub-heading has to stand alone as instructions.  Can't rely on referring up, or assuming the reader has read or retained earlier information.